chatbot/ai-studio.yaml

```diff
@@ -6,6 +6,9 @@ application:
     - name: chatbot-inference-app
       contextdir: ai_applications
       containerfile: builds/Containerfile
+      arch:
```

Comment:
I don't think we need an arch field here

Reply:
I don't have a strong opinion on it, so i added it on Sally's suggestion but ...can there be a case where the sample app has two version, one for amd and one for arm? In that case it may be useful. Or we could state that if there is no arch field is good for any arch.

Reply:
Yes my comment is just as the client application is a full Python app it has no arch adherence

Reply:
@jeffmaury removed the last commit having the update.